aws/cloudformation/standalone/greenhouse_dns.yml

```diff
@@ -0,0 +1,48 @@
+# Greenhouse DNS Configuration
+# See: https://support.greenhouse.io/hc/en-us/articles/201111684-Email-domain-verification
+
+Resources:
+
+  GreenhouseHostedZone:
+    Type: "AWS::Route53::HostedZone"
+    Properties:
+      Name: 'gh-mail.code.org'
+
+  NameServerRecord:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      HostedZoneName : code.org.
+      Name: gh-mail.code.org
```

Comment:
Could we avoid using an acronym for this domain name? Would a longer `greenhouse-mail.code.org` negatively impact us?

Reply:
Nope, this is a greenhouse requirement :(